workout-tracker/src/components/MainNavBar/MainNavBar.js

```diff
@@ -18,6 +18,11 @@ const MainNavBar = (props) => (
     <div className="spacer" />
     <ul className="main-nav__items">
       <NavigationItems isAuth={props.isAuth} onLogout={props.onLogout} />
+      {props.isAuth && (
+    <li className="navigation-item" key="logout">
```

Comment:
This is actually a trifle. Could you use `__` for all class naming or  just use `-` instead of both.  Looking for consistency here.

Reply:
`__` indicates that the html is a child component.
i.e. See excerpt of code below
<button className="mobile-toggle" onClick={props.onOpen}>
    <span className="mobile-toggle__bar" />
    <span className="mobile-toggle__bar" />
    <span className="mobile-toggle__bar" />
  </button>
Thus I know that __bar is a child component of mobile-toggle.

Reply:
Oh, pardon my ignorance.